cmd/api-linter/cli.go

```diff
@@ -60,6 +62,10 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 					Value: ".",
 					Usage: "the directory in which for protoc to search for imports",
 				},
+				cli.BoolTFlag{
```

Comment:
I don't think this should default to true.

The normal output of this CLI is structured format (yaml or json) which can be piped into another process for further processing. By default, adding tabular, freeform text could break that.